unstoppable-ios-app/domains-manager-ios/Services/WalletConnect/WalletConnectService_V2/WalletConnectServiceV2.swift

```diff
@@ -292,7 +292,7 @@ class WalletConnectServiceV2: WalletConnectServiceV2Protocol, WalletConnectV2Pub
     }
     
     private func canSupport( _ proposal: SessionV2.Proposal) -> Bool {
-        guard proposal.requiredNamespaces.count == 1 else { return false }
+        guard proposal.requiredNamespaces.count > 1 else { return true }
```

Comment:
Before it should've have one namespace, otherwise is not supported. 
Not if there's 0 or 1 - it is supported. What is the logic behind it?

Reply:
Long read: https://docs.google.com/document/d/1kIwtpqU8zCvsSeCdKkUF3kJTsJYHss85rjaSqukVRSM/edit
In short: if the required namespace is undefined we should definitely connect